youtube_dl/downloader/rtmp.py

```diff
@@ -143,6 +144,8 @@ def run_rtmpdump(args):
             basic_args += ['--conn', conn]
         if protocol is not None:
             basic_args += ['--protocol', protocol]
+        if real_time:
+            basic_args += ['--realtime']
```

Comment:
> Works perfectly without --realtime for me, downloaded video is fine. Is it really required?

It is like a scratched vinyl record: at some point you see block artifacts and it jumps back in time. At first glance the video looks fine, you have to watch a few seconds to notice this. I would expect this problem to be more pronounced for longer videos, but the one from the test case is also affected.